www/js/app.js

```diff
@@ -145,6 +145,32 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('searchArticles').click();
         return false;
     });
+     //Handle Home key press to focus #prefix
+     $(window,document).on('keydown', function(e) {
+         //if home key is pressed then
+         if(/^Home/.test(e.key)) {
+             //scroll to top of #search-article section since scroll bar is associated with it and not to window or document
+             $('#search-article').scrollTop(0);
+             $('#prefix').focus();
+         }
+     });
+     //Handle Home key event when focus is inside iframe #articleContent
+     var iframe= document.getElementById('articleContent');
```

Comment:
Be careful to not collide with another `iframe` variable (like the one in `loadCSSJQuery`, that is not the same).